creator/ingest_runs/models.py

```diff
@@ -141,6 +144,13 @@ def get_job_queue(**kwargs):
         kwargs["name"] = INGEST_QUEUE_NAME
         return django_rq.get_queue(**kwargs)
 
+    @property
```

Comment:
You can remove `get_job_queue` now that you have `queue` property

Reply:
oops!